docs/integration/integration-more.md

```diff
@@ -0,0 +1,163 @@
+---
+sidebar_position: 60
+title: More and Troubleshooting
+sidebar_label: More and Troubleshooting
+---
+
+
+## Coin name and ticker
+
+* Name: Alephium
+* Ticker/Symbol: ALPH
+
+## Explorer URLs
+
+* Mainnet: https://explorer.alephium.org
+* Testnet: https://testnet.alephium.org/
+
+## Public API interface URL
+
+URL: https://wallet-v20.mainnet.alephium.org
+
+All public APIs are listed and updated on the [Public services](../dapps/public-services.md) page.
+
+## Official wallets
+
+Official desktop wallet: https://github.com/alephium/desktop-wallet/releases/latest
```

Comment:
Probably good to use links, feels cleaner. E.g.

[Android](https://play.google.com/store/apps/details?id=org.alephium.wallet)